doc/api-extensions.md

```diff
@@ -2285,3 +2285,6 @@ This parameter allows bypassing the project name filter.
 ## `metadata_configuration`
 Adds the `GET /1.0/metadata/configuration` API endpoint to retrieve the generated metadata configuration in a JSON format. The JSON structure adopts the structure ```"configs" > `ENTITY` > `ENTITY_SECTION` > "keys" > [<CONFIG_OPTION_0>, <CONFIG_OPTION_1>, ...]```.
 Check the list of {doc}`configuration options </config-options>` to see which configuration options are included.
+
+## `full_documentation`
```

Comment:
Did we have an API extension for `/ui`? If not then we dont need one for this, as its not really part of the API.